lib/store/ordersSlice.ts

```diff
@@ -0,0 +1,55 @@
+import { StateCreator, StoreApi, UseBoundStore, create } from "zustand";
+import { BoardOrders, StateOrdersUpdate, VehicleOrders } from "..";
+
+export interface OrdersSlice {
+    vehicleOrders: VehicleOrders
+    setOrders: (vehicleOrders: VehicleOrders) => void
+    updateStateOrders: (stateOrdersUpdate: StateOrdersUpdate) => void
+}
+
+export const ordersSlice: StateCreator<OrdersSlice> = (set, get) => ({
+    vehicleOrders: { boards: [] },
+
+    /**
+     * Reducer that sets the vehicleOrders to the vehicleOrders param
+     * @param {VehicleOrders} vehicleOrders
+     */
+    setOrders: (vehicleOrders: VehicleOrders) => {
+        set(state => ({
+            ...state,
+            vehicleOrders: vehicleOrders
+        }))
+    },
+
+    /**
+     * Reducer that updates orders to stateOrdersUpdate param.
+     * It checks if the board vinculated with each order of stateOrdersUpdate exists.
+     * If so, it is updated. If not exists, it ignores that order.
+     * @param {StateOrdersUpdate} stateOrdersUpdate 
+     */
+    updateStateOrders: (stateOrdersUpdate: StateOrdersUpdate) => {
+        Object.entries(stateOrdersUpdate).forEach(([name, ids]) => {
+            const index = get().vehicleOrders.boards.findIndex( (board) => board.name == name );
+            if (index == -1) return
```

Comment:
I feel we need to do all of this because our data structure is not the best, try to think about better ways so we avoid doing all these checks and search each time we receive data and write them down somewhere (maybe a feature issue or in software-docs)